docs/volto/bootstrap.md

```diff
@@ -24,63 +24,84 @@ Make sure you set a CORS policy or things tend to magically go wrong. See <https
 
 (install-deps-volto-label)=
 
-## Installing Dependencies
+## Installing Prerequisites
 
-First step is to install the correct Node version using `nvm`:
+First step is to install the correct Node version using `nvm`, as indicated [here](https://6.dev-docs.plone.org/volto/getting-started/install.html#install-nvm-nodejs-version-manager).
 
-```shell
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
-```
-
-Then you can install the latest LTS version of node:
 
-```shell
-nvm install lts/fermium
-nvm use lts/fermium
-```
+Install `Yarn` as indicated [here](https://6.dev-docs.plone.org/volto/getting-started/install.html#yarn-nodejs-package-manager).
```

Comment:
See previous comment.

Also in this context, Yarn is a proper noun, not a command or inline literal.

```suggestion
[Install Yarn](https://6.dev-docs.plone.org/volto/getting-started/install.html#yarn-nodejs-package-manager).
```

Reply:
Done.